src/layout/_template.njk

```diff
@@ -44,6 +42,15 @@
     {% set page_title = "ONS Design System" %}
 {% endif %}
 
+{# Page columns #}
+{% set mainColNumber = pageConfig.mainCol.columns | default("8") %}
+{% set mainColClasses = (' ' + pageConfig.mainCol.colClasses) if pageConfig.mainCol is defined and pageConfig.mainCol or pageConfig.asideCol.position is defined and pageConfig.asideCol.position == "before" %}
```

Comment:
`or pageConfig.asideCol.position is defined and pageConfig.asideCol.position == "before"` not sure why this is in this clause? We are setting the class to be empty if the position of the aside is "before"

Reply:
Also, we are requiring that `mainCol.colClasses` exists if `asideCol.position == before`

Reply:
Left in by mistake.